### PR TITLE
focus for intro to graphs

### PR DESCRIPTION
**Pull request by ryanchou-dev**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

File: content/2_Bronze/Intro_Graphs.mdx [hunk 1]
```diff
@@ -71,6 +71,408 @@ categories:
...
+	output_file.write("\n")
+```
+</PySection>
+</LanguageSection>
+
+
+</Spoiler>
+
+Credit to: Neo Wang, Aadit Ambadkar, Sandeep Wadhwa, Kevin Sheng, Nikhil Chatterjee
+
```
Comment (SansPapyrus683) [inline content/2_Bronze/Intro_Graphs.mdx:475]: put this in the author/contributor thing instead ig

File: content/2_Bronze/Intro_Graphs.mdx [hunk 2]
```diff
@@ -71,6 +71,408 @@ categories:
 
 Knowing [DFS](/silver/dfs) can be helpful but it should not be required.
 
+## Example - Grass Planting
+
+<FocusProblem problem="tutorial" />
+
+<Spoiler title="Hint 1">
+Find a lower bound for the answer, and try to prove it!
+</Spoiler>
```
Comment (SansPapyrus683) [inline content/2_Bronze/Intro_Graphs.mdx:80]: ngl this hint is...kinda weird

Comment (SansPapyrus683) [reply content/2_Bronze/Intro_Graphs.mdx:80]: @envyaims whatre ur thoughts

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

**Comment by SansPapyrus683** [comment]
actually i'd uh remove the vid sol too ryan

**Comment by ryanchou-dev** [comment]
fixes #3541

File: content/2_Bronze/Intro_Graphs.mdx [hunk 3]
```diff
@@ -71,6 +71,316 @@ categories:
...
+
+</PySection>
+</LanguageSection>
+</Spoiler>
+
+
+<Spoiler title="Alternate Solution">
+**Time Complexity**: $\mathcal{O}(N)$
+
+This can also be solved using [DFS](/silver/dfs).
```
Comment (bqi343) [inline content/2_Bronze/Intro_Graphs.mdx:189]: this problem probably shouldn't be in the Bronze module

Comment (ryanchou-dev) [reply content/2_Bronze/Intro_Graphs.mdx:189]: ack ok

**Comment by ryanchou-dev** [comment]
will redo with diff prob later

**Pull request closed without merging**